src/spectral_recovery/enums.py

```diff
@@ -43,3 +43,11 @@ class Metric(Enum):
 
     def __str__(self) -> str:
         return self.value
+
+
+class Platform(Enum):
+    # NOTE: do we want to support Landsat_MSS?
```

Comment:
I think we can probably avoid MSS for now at least, due to the coarser spectral resolution. Probably best to focus on the higher resolution sensors for now.